python/cudf/cudf/core/column/categorical.py

```diff
@@ -1236,9 +1236,8 @@ def notnull(self) -> ColumnBase:
     def fillna(
         self,
         fill_value: Any = None,
-        method: Any = None,
-        dtype: Optional[Dtype] = None,
-    ) -> CategoricalColumn:
+        method: Optional[str] = None,
```

Comment:
Just checking -- this is non-public so the API break is not a "breaking change" right?

Reply:
I've been working under that assumption that cudf column objects are not public cc @shwina

Reply:
Yes, that should be fine! Just wanted to check in case this interacts with any future changes you were planning for public types.

Reply:
Gotcha. Yeah the arguments being modified in `Column.fillna` do not clash with `Series/DataFrame.fillna`